src/main.js

```diff
@@ -367,6 +368,14 @@ app.on('ready', () => {
   if (global.willAppQuit) {
     return;
   }
+  if (buildConfig.enableServerManagement === false && buildConfig.defaultTeams.length === 0) {
+    dialog.showMessageBox({
+      type: 'error',
+      title: 'Mattermost',
+      message: 'When "enableServerManagement: false" is specified in buildConfig.js,\n"defaultTeams" must have one team at least.'
```

Comment:
Do you think we need the UI dialog now that we automatically message the admin? Or does it still build the app files in which case there's a chance the app will be distributed to others?

Reply:
Surely we added the automatic checker to the build script. However by reading the contents of `package.json`, admin can distribute the app with wrong buildConfig. I think there is no such admin, though.

So I'm okay even if the UI dialog is removed. I could not determine whether the dialog is still needed.

Reply:
Wondering if `npm run package:*` fail if the admin is distributing the app with an incorrect buildConfig? Thoughts? (Not sure if that's even feasible)

Reply:
Yes. When using wrong buildConfig, `npm run package:*` fails.

Actually, `npm run package:*` consists from three steps.

1. Check buildConfig. (this PR)
2. Compile JavaScript with webpack.
3. Create distributable packages with electron-builder.

The step 3 is written at `package.json`, so the admin can create packages without error by manually executing only step 3. However it's intentional. So I think it's okay even if we ignore such case.

Reply:
Sounds great. If people hit issues, we can always add the dialog later, but sounds like this should be okay for the majority of the admins.

We can remove the dialog 👍